common/app/common/JsMinifier.scala

```diff
@@ -19,11 +19,10 @@ object JsMinifier {
     /* Checks */
     options.setCheckTypes(true)
     options.setCheckSuspiciousCode(true)
-    options.setReportMissingOverride(CheckLevel.WARNING)
-    options.setCheckProvides(CheckLevel.WARNING)
-    options.setCheckGlobalNamesLevel(CheckLevel.WARNING)
```

Comment:
Deleted in https://github.com/google/closure-compiler/commit/f8d26da9ca0b9444524c5be650981c97c369bc56